shell/browser/native_window.h

```diff
@@ -70,7 +70,8 @@ class NativeWindow : public base::SupportsUserData,
   virtual void Close() = 0;
   virtual void CloseImmediately() = 0;
   virtual bool IsClosed() const;
-  virtual void Focus(bool focus) = 0;
+  virtual void Focus(const gin_helper::Dictionary& options) = 0;
```

Comment:
I think this should be a strict struct that we pass around rather than a "Dictionary" which is basically a wildcard.  I'm thinking just a `FocusOptions` struct.

In general I think we try to draw the line that `api_*` can use gin and v8 and the other things (in this case `native_window`) should use actual structs and such.

Reply:
Makes sense to me. Can I just define this in `native_window.h` or is there a better place for it?

Reply:
@tonyferrell  `native_window.h` makes sense I think 👍